utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/encode/JsonSchemaOutputValidator.scala

```diff
@@ -44,9 +44,6 @@ class JsonSchemaOutputValidator(validationMode: ValidationMode) extends LazyLogg
         validateUnionInput(union, schema, path)
       case (typingResult: TypedObjectTypingResult, s: ObjectSchema) =>
         validateRecordSchema(typingResult, s, path)
-      case (_@TypedNull, _) if !schema.isNullable =>
-        invalid(typingResult, schema, path)
```

Comment:
Right now we support union schema, and some primitive validation for it, is done at `canBeSubclassOf`